truenas_install/__main__.py

```diff
@@ -306,7 +306,10 @@ def main():
                         # Set bootfs before running update-grub
                         run_command(["zpool", "set", f"bootfs={dataset_name}", pool_name])
 
-                        run_command([f"{root}/usr/local/bin/initrd-zfs.py", pool_name, root, "0"])
+                        cp = run_command([f"{root}/usr/local/bin/truenas-initrd.py", root], check=False)
+                        if cp.returncode > 1:
```

Comment:
Why > 1?

Reply:
Exit status of `0` indicates that initramfs was not updated and no reboot is required. If it's `1` it means it was updated and a reboot is required and bigger then `1` means script errored out.